pkg/controller/actions/deploy/action_deploy_merge_deployment.go

```diff
@@ -106,3 +107,29 @@ func MergeDeployments(source *unstructured.Unstructured, target *unstructured.Un
 
 	return nil
 }
+
+// for the case: in manifests, Recreate is used as the target and rollingupdate is set, we remove it totally.
+func ConvertRollingUpdate(source, target map[string]interface{}) error {
```

Comment:
maybe use unstructured here so the intent is clear

Reply:
yep, updated